src/main/java/com/sumologic/log4j/aggregation/SumoBufferFlusher.java

```diff
@@ -86,6 +86,10 @@ public void stop() {
         if (executor != null) {
             executor.shutdownNow();
         }
+
+        // flush last time before dying
+        flushingTask.setMessagesPerRequest(1L);
```

Comment:
maybe leave a comment that this is to satisfy `needsFlushing` in `BufferFlushingTask` (wasn't clear to me at first)